mapfishapp/src/main/webapp/app/js/GEOR_ResultsPanel.js

```diff
@@ -61,6 +61,9 @@ GEOR.ResultsPanel = Ext.extend(Ext.Panel, {
      */
     sfControl: null,
 
+    //TODO doc
+    id: "resultPanel",
```

Comment:
Really not sure this is a good idea.
ids are autogenerated by ExtJS for each Panel instance